src/main/java/aquality/appium/mobile/configuration/IConfigurationsModule.java

```diff
@@ -0,0 +1,22 @@
+package aquality.appium.mobile.configuration;
+
+/**
+ * Describes implementations of configurations to be registered in DI container.
+ */
+public interface IConfigurationsModule extends aquality.selenium.core.configurations.IConfigurationsModule {
+
+    /**
+     * @return class which implements {@link IApplicationProfile}
+     */
+    default Class<? extends IApplicationProfile> getBrowserProfileImplementation() {
```

Comment:
shouldn't it be renamed as get**APPLICATION**ProfileImplementation?

Reply:
OFC, thx